docker/docker-compose.yml

```diff
@@ -208,8 +208,8 @@ services:
     container_name: mainflux-vernemq
     restart: on-failure
     environment:
-      DOCKER_VERNEMQ_ALLOW_ANONYMOUS: "on"
-      DOCKER_VERNEMQ_LOG__CONSOLE__LEVEL: error
+      DOCKER_VERNEMQ_ALLOW_ANONYMOUS: ${MF_DOCKER_VERNEMQ_ALLOW_ANONYMOUS}
```

Comment:
Would ommiting `""` here create problem? I guess not, just asking if this was tested?

Reply:
AFAIK for docker-compose it's the same

Reply:
This will never be `off` because Manflux doesn't use Broker authentication. IMHO no need for env var.